lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -8,57 +8,90 @@
 @import '../../scss-shared/nx-text-helpers';
 @import '../../scss-shared/nx-container-helpers';
 
+$nx-dropdown-width: 250px;
+
 .nx-dropdown {
   display: inline-block;
   position: relative;
+  width: $nx-dropdown-width;
 }
 
 .nx-dropdown__toggle {
+  display: flex;
   margin: 0;
   outline: none;
+  text-align: left;
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
+  width: $nx-dropdown-width;
+
+  .nx-dropdown__toggle-label {
+    flex-grow: 1;
+  }
+
+  .nx-icon {
+    align-self: center;
+    color: #000000;
+  }
+
+  &.disabled {
+    .nx-icon {
+      color: #949494;
+    }
+  }
+
+  &.open {
+    border-color: #045dd1;
+  }
 }
 
 .nx-dropdown-menu {
   @include container-vertical();
 
   background: $nx-off-white;
   border: $nx-border-dark;
-  border-radius: 4px;
-  box-shadow: -4px 4px 10px 0 $nx-slate;
+  border-radius: $nx-border-radius;
+  box-shadow: 0 2px 12px 0 rgba(74, 77, 113, 0.2),
+              0 1px 2px 0 rgba(91, 95, 141, 0.2),
+              0 1px 1px 0 rgba(124, 128, 179, 0.2);
   color: $nx-text-color-light;
   display: block;
-  margin: 6px 0 0 0;
-  max-width: 300px;
-  min-width: 160px;
-  padding: 8px 12px;
+  max-height: 328px;
+  margin: $nx-spacing-xxs 0 0 0;
+  overflow-y: auto;
+  padding: $nx-spacing-xxs 0;
   position: absolute;
   top: 100%;
+  width: $nx-dropdown-width;
```

Comment:
This results in it actually being 2px wider due to not having `box-sizing: border-box`.  Separately, you could just use `width: 100%` here.

Reply:
> This results in it actually being 2px wider due to not having `box-sizing: border-box`. Separately, you could just use `width: 100%` here.
I put `border-box` on it which then caused me to have to undo the width change, but I prefer keeping the widths in sync with the variable: https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/45c93280c46314b8e35ae8512c23559c088214d5